pytest_catchlog.py

```diff
@@ -122,6 +134,15 @@ def __init__(self, config):
         self.formatter = logging.Formatter(
                 get_option_ini(config, 'log_format'),
                 get_option_ini(config, 'log_date_format'))
+        terminal = py.io.TerminalWriter(sys.stderr)  # pylint: disable=no-member
```

Comment:
I wrote that part a while back, In don't remember why now. I can switch that to sys.stderr if you prefer....